crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -946,6 +954,15 @@ pub struct RoomInfo {
     /// filled at start when creating a room, or on every successful sync.
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub(crate) cached_display_name: Option<DisplayName>,
+
+    /// The recency timestamp of this room.
+    ///
+    /// It's not to be confused with `origin_server_ts` of the latest event.
+    /// Sliding Sync might "ignore” some events when computing the recency
+    /// timestamp of the room. Thus, using this `recency_timestamp` value is
+    /// more accurate than relying on the latest event.
+    #[cfg(feature = "experimental-sliding-sync")]
+    pub(crate) recency_timestamp: Option<MilliSecondsSinceUnixEpoch>,
```

Comment:
nit: add a `#[serde(default)]` annotation, otherwise `RoomInfo`s serialized before this patch will fail to deserialize, resulting in a global error that may lead to user logouts on EX apps.

Reply:
Oh, nice, thanks!